### PR TITLE
Fix #811 - fetch and show emoji icons

### LINKED ISSUE #811: Problems with Twitter spout for accounts with emojis
For example, the tweet https://twitter.com/archiloque/status/794160285615616000 

captura de pantalla de 2016-11-03 10 26 44

appears the following way in selfoss:

captura de pantalla de 2016-11-03 10 27 26

### PR DESCRIPTION
**Pull request by severo**
See https://github.com/SSilence/selfoss/issues/811 and https://github.com/SSilence/selfoss/issues/534#issuecomment-172329123

### CODE REVIEW

File: libs/f3/db/sql.php [hunk 1]
```diff
@@ -461,8 +461,7 @@ function __construct($dsn,$user=NULL,$pw=NULL,array $options=NULL) {
 		if (!$options)
 			$options=array();
 		if (isset($parts[0]) && strstr($parts[0],':',TRUE)=='mysql')
-			$options+=array(\PDO::MYSQL_ATTR_INIT_COMMAND=>'SET NAMES '.
-				strtolower(str_replace('-','',$fw->get('ENCODING'))).';');
+			$options+=array(\PDO::MYSQL_ATTR_INIT_COMMAND=>'SET NAMES utf8mb4;');
```
Comment (jtojnar) [inline libs/f3/db/sql.php]: This will break when we update f3 framework. Better raise issue [there](https://github.com/bcosca/fatfree/issues/).

Comment (severo) [inline libs/f3/db/sql.php]: OK, thanks. I created the following issue, but I don't think I can help more on this problem.

https://github.com/bcosca/fatfree/issues/976

**Comment by jtojnar** [comment]
Also migration script will probably be needed, otherwise users will receive the same error you got here https://github.com/SSilence/selfoss/issues/811#issuecomment-258187844, just with charsets reversed.

**Comment by jtojnar** [comment]
@SSilence this is not ready to be merged, it will break existing installations. We need migration script first.

**Comment by jtojnar** [comment]
@SSilence I can add it though.

**Comment by severo** [comment]
It would be wonderful! I'm afraid I would last a lot before doing it...

**Comment by jtojnar** [comment]
@SSilence I introduced the upgrade script in #819

**Pull request merged**